interface/billing/sl_eob_search.php

```diff
@@ -166,10 +166,13 @@ function upload_file_to_client($file_to_send) {
   // sleep one second to ensure there's no follow-on.
   sleep(1);
 }
-function upload_file_to_client_pdf($file_to_send) {
+function upload_file_to_client_pdf($file_to_send, $aPatFirstName = '', $aPatID = null, $flagCFN = false) { //modified for statement title name
   //Function reads a HTML file and converts to pdf.
 
-  global $STMT_TEMP_FILE_PDF;
+   $aPatFName = preg_replace("/[^A-Za-z]/", '', $aPatFirstName); //modified for statement title name
+   if($flagCFN) $STMT_TEMP_FILE_PDF = $GLOBALS['temporary_files_dir'] . "/Stmt_{$aPatFName}_{$aPatID}.pdf";
+   else global $STMT_TEMP_FILE_PDF;
```

Comment:
Lets make this if/else follow psr2:
http://www.php-fig.org/psr/psr-2/#if-elseif-else
``````
if ($flagCFN) {
    $STMT_TEMP_FILE_PDF = $GLOBALS['temporary_files_dir'] . "/Stmt_{$aPatFName}_{$aPatID}.pdf";
} else {
    global $STMT_TEMP_FILE_PDF;
}